### DISCUSSION TITLE
One line Form & Entity Title

### DISCUSSION THREAD

**Discussion by mpchean** [burkybang/SuiteAdvanced-Feedback#220] [Ideas]
Move Entity title to the same line as the Form Title and freeze that line while you scroll so you always know where you are and who you are working on.

**Comment by burkybang** [comment]
Can you provide a screenshot to help me understand your idea?

**Reply by mpchean** [reply]
I'll try - I'm not a web person but I will see what I can work up.  Btw wrote about you on the Netsuite Reddit forum.  Netsuite should have hired you to rewrite their UI, so much better than Redwood!

**Reply by mpchean** [reply]
![form](https://github.com/user-attachments/assets/04be275c-26b6-4010-9fd8-c52b428a5612)

**Reply by burkybang** [reply]
Oh thanks!

I have a solution to your suggestion already. The tab title displays that information, which would also be available as you scroll.

**Reply by mpchean** [reply]
Thanks didn't notice that.  Very nice.
